app/assets/stylesheets/modules/_family.scss

```diff
@@ -0,0 +1,151 @@
+.family {
+  background-color: $white;
+  border-radius: $base-border-radius;
+  margin-left: 2.5%;
+  padding: em(40);
+  width: 95%;
+
+  @include media($medium-screen-up) {
+    @include span-columns(10 of 12);
+    @include shift(1);
+
+    padding: em(90);
+  }
+
+  &-section-one {
+
+    &-content {
+      @include span-columns(10 0f 10);
+
+      @include media($medium-screen-up) {
+        @include span-columns(6.8 of 10);
+
+        margin-left: em(10);
+      }
+    }
+
+    &-image {
+      @include span-columns(12 of 12);
+
+      background-repeat: no-repeat;
+      background-size: 100%;
+      background: image-url("julian-michelle.png");
+      display: block;
+      height: em(200);
+
+      @include media($medium-screen-up) {
+        @include span-columns(3 of 10);
+
+        background-size: 100%;
+        display: inline-block;
+        height: em(120);
+      }
+
+    }
+
+  }
+
+  &-section-two-image {
+    background-repeat: no-repeat;
```

Comment:
Properties should be ordered background, background-repeat, background-size, display, height, margin, width